test/system/deposit-liquidated-externally.test.js

```diff
@@ -0,0 +1,161 @@
+  let bidder
+
+  before(async () => {
+    await resetFork(startingBlock)
+
+    governance = await ethers.getSigner(0)
+    rewardsManager = await ethers.getSigner(1)
+
+    tbtcToken = await ethers.getContractAt("IERC20", tbtcTokenAddress)
+
+    const UnderwriterToken = await ethers.getContractFactory("UnderwriterToken")
+    underwriterToken = await UnderwriterToken.deploy("Coverage KEEP", "covKEEP")
+    await underwriterToken.deployed()
+
+    const AssetPool = await ethers.getContractFactory("AssetPool")
+    assetPool = await AssetPool.deploy(
+      keepTokenAddress,
+      underwriterToken.address,
+      rewardsManager.address
+    )
+    await assetPool.deployed()
+    await underwriterToken.transferOwnership(assetPool.address)
+
+    const SignerBondsSwapStrategy = await ethers.getContractFactory(
+      "SignerBondsEscrow"
+    )
+    signerBondsSwapStrategy = await SignerBondsSwapStrategy.deploy()
+    await signerBondsSwapStrategy.deployed()
+
+    const Auction = await ethers.getContractFactory("Auction")
+
+    const masterAuction = await Auction.deploy()
+    await masterAuction.deployed()
+
+    const CoveragePool = await ethers.getContractFactory("CoveragePool")
+    coveragePool = await CoveragePool.deploy(assetPool.address)
+    await coveragePool.deployed()
+    await assetPool.transferOwnership(coveragePool.address)
+
+    const RiskManagerV1 = await ethers.getContractFactory("RiskManagerV1")
+    riskManagerV1 = await RiskManagerV1.deploy(
+      tbtcToken.address,
+      tbtcDepositTokenAddress,
+      coveragePool.address,
+      signerBondsSwapStrategy.address,
+      masterAuction.address,
+      auctionLength,
+      bondAuctionThreshold
+    )
+    await riskManagerV1.deployed()
+
+    await coveragePool
+      .connect(governance)
+      .approveFirstRiskManager(riskManagerV1.address)
+
+    tbtcDeposit = await ethers.getContractAt("IDeposit", depositAddress)
+    bidder = await impersonateAccount(bidderAddress)
+  })
+
+  describe("test initial state", () => {
+    describe("deposit", () => {
+      it("should be in active state", async () => {
+        expect(await tbtcDeposit.currentState()).to.equal(5) // Active
+      })
+    })
+
+    describe("auction", () => {
+      it("should not exist", async () => {
+        const auctionAddress = await riskManagerV1.depositToAuction(
+          tbtcDeposit.address
+        )
+        expect(auctionAddress).to.be.equal(ZERO_ADDRESS)
+      })
+    })
+  })
+
+  describe("when deposit has been liquidated outside Coverage Pools", () => {
+    let auction
+
+    before(async () => {
+      await tbtcDeposit.notifyRedemptionSignatureTimedOut()
+      // We need additional 9% to pass the risk manager threshold. To get this
+      // part, we need 22870 seconds to elapse. This is because the auction
+      // length is 86400 seconds (24h) and there is 34% of bonds remaining.
+      // So, additional 9% will be offered after 9/34 * 86400s.
+      await increaseTime(22870)
```

Comment:
I think we can get rid of this by decreasing the `bondAuctionThreshold` to a value less than `66%`. This code originates from the `system/liquidation.test.js` when we test the risk manager threshold logic by the way. Because we covered that there, I don't think there is a need to repeat that in all test cases. WDYT?

Reply:
Side note, I think we should explain in English why do we modify the initial state forked from the chain and what we are trying to achieve this way in the test, and why. This would help to catch issues like the one Lukasz described.

Reply:
I decreased the value of `bondAuctionThreshold` and removed `bondAuctionThreshold`: 220522bf9e4417ee41ee85570e359b86931a0a48.